tests/test_channel_metrics/test_completely_bounded_trace_norm.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for completely_bounded_trace_norm."""
+import numpy as np
+
+from toqito.channels.dephasing import dephasing
+from toqito.channel_metrics.completely_bounded_trace_norm import completely_bounded_trace_norm
```

Comment:
@vprusso I made this alteration, but this import seems not to work, as not the function but a module is imported. Should I change it to the prior version, or is sth wrong with the init?

Reply:
Looks like the `__init__.py` is incorrect. For clarity, the `__init__.py` will look like this:

```py
from toqito.channel_metrics.channel_fidelity import channel_fidelity
from toqito.channel_metrics.diamond_norm import diamond_norm
from toqito.channel_metrics.completely_bounded_spectral_norm import completely_bounded_spectral_norm
from toqito.channel_metrics.completely_bounded_trace_norm import completely_bounded_trace_norm
```

Then, if you want to refer to these functions in another file, you can write the more condensed:

```py
from toqito.channel_metrics import X
```

instead of the longer statement

```py
from toqito.channel_metrics.X import X
```

Does that make sense?

Reply:
changed it accordingly @vprusso , I though the init could be made also more concise, but should be fine